org.virt_manager.virt-viewer.yml

```diff
@@ -0,0 +1,186 @@
+        sha256: 0b0e3aa07c8c063ddf40b082bdf7e37a1562bda40a0ff5272957f3e987e0e54b
+  - name: gtk-vnc
+    buildsystem: meson
+    cleanup:
+      - /bin
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/gtk-vnc/1.3/gtk-vnc-1.3.1.tar.xz
+        sha256: 512763ac4e0559d0158b6682ca5dd1a3bd633f082f5e4349d7158e6b5f80f1ce
+  - name: libvirt-glib
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://libvirt.org/sources/glib/libvirt-glib-4.0.0.tar.xz
+        sha256: 8423f7069daa476307321d1c11e2ecc285340cd32ca9fc05207762843edeacbd
+    modules:
+      - name: libnl
+        config-opts:
+        - "--enable-cli=no"
+        sources:
+          - type: archive
+            url: >-
+              https://github.com/thom311/libnl/releases/download/libnl3_5_0/libnl-3.5.0.tar.gz
+            sha256: 352133ec9545da76f77e70ccb48c9d7e5324d67f6474744647a7ed382b5e05fa
+      - name: portablexdr
+        buildsystem: autotools
+        config-opts:
+          - "--disable-static"
+        sources:
+          - type: archive
+            url: >-
+              https://people.redhat.com/~rjones/portablexdr/files/portablexdr-4.9.1.tar.gz
+            sha256: 5cf4bdd153cf4d44eaf10b725f451d0cfadc070b4b9a9ccfb64094b8f78de72c
+          - type: script
+            dest-filename: autogen.sh
+            commands:
+              - AUTOMAKE="automake --foreign" autoreconf -vfi
+          - type: shell
+            commands:
+              - cp -f /usr/share/gnu-config/config.sub .
+              - cp -f /usr/share/gnu-config/config.guess .
+      - name: libtirpc
+        config-opts:
+          - "--disable-gssapi"
+          - "--libdir=/app/lib"
+        sources:
+          - type: archive
+            url: >-
+              https://downloads.sourceforge.net/sourceforge/libtirpc/libtirpc-1.3.2.tar.bz2
+            sha256: e24eb88b8ce7db3b7ca6eb80115dd1284abc5ec32a8deccfed2224fc2532b9fd
+      - name: rpcsvc-proto
+        buildsystem: autotools
+        sources:
+          - type: archive
+            url: https://github.com/thkukuk/rpcsvc-proto/archive/v1.4.1.tar.gz
+            sha256: 750f7e57b81407a25b707867e90d7ee80aeb53bf515b114fc218f3c78dc9a6e8
+      - name: libvirt
+        builddir: true
+        buildsystem: meson
+        config-opts:
+          - "-Dbash_completion=disabled"
+          - "-Dtests=disabled"
+          - "-Ddocs=disabled"
+          - "-Dstorage_dir=disabled"
+          - "-Dstorage_fs=disabled"
+          - "-Dstorage_lvm=disabled"
+          - "-Dstorage_iscsi=disabled"
+          - "-Dstorage_scsi=disabled"
+          - "-Dstorage_mpath=disabled"
+          - "-Dstorage_disk=disabled"
+          - "-Dstorage_rbd=disabled"
+          - "-Dstorage_sheepdog=disabled"
+          - "-Dstorage_gluster=disabled"
+          - "-Dstorage_zfs=disabled"
+          - "-Dstorage_vstorage=disabled"
+        build-options:
+          cflags: "-I/app/include/tirpc -ltirpc"
```

Comment:
I would not expect this to be required.  libtirpc publishes a pkg-config file, which libvirt should discover automatically.